crates/bevy_reflect/src/lib.rs

```diff
@@ -538,14 +535,15 @@ pub mod __macro_exports {
 #[cfg(test)]
 #[allow(clippy::disallowed_types, clippy::approx_constant)]
 mod tests {
-    #[cfg(feature = "glam")]
-    use ::glam::{quat, vec3, Quat, Vec3};
-    use ::serde::{de::DeserializeSeed, Deserialize, Serialize};
     use bevy_utils::HashMap;
+    #[rustfmt::skip]
+    use ::glam::{quat, vec3, Quat, Vec3};
```

Comment:
Could you elaborate why this is needed?

Reply:
This one probably isn't, adding glam as a dev-dep should take care of it, I'll remove it.

Reply:
This does seem to be required still, later comments cover this topic.